scripts/jenkins/jobs-ibs/cloud-mkcloud7.yaml

```diff
@@ -13,6 +13,7 @@
     label: openstack-mkcloud-SLE12-x86_64
     sles12sp2: 0
     jobs:
+        - 'cloud-mkcloud{version}-job-96compute-{arch}'
```

Comment:
as that job can no be run on the openstack-mkcloud-SLE12-x86_64 workers, you need to add a separate section for running the job on a "openstack-mkcloud-SLE12-bigmachine-x86_64" (or something like that).